HW12_1.py

```diff
@@ -0,0 +1,72 @@
+phone_book = {
+    'Igor': 132434,
+    'Diana': 254345,
+}
+
+import json
+
+json_data = json.dumps(phone_book)
+with open (r'C:\01_Education\03_IT\04_Pyhon\Laba\HW\phone_book.txt', 'a') as file:
+    file.write(json_data)
+
+print (phone_book.keys(),phone_book.values())
+
+while True:
+    user_input = input('Enter a command: ')
+    split_input = user_input.split()
+   
+    command = split_input[0]
+    
+    if len(split_input) > 1:
+        name = split_input[1]
+    else:
+        name = None
+
+    if len(split_input) > 2:
+        number = split_input[2]
+    else:
+        number = None
+    
+    if command == 'stats':
+        print(len(phone_book))
+
+    elif command == 'add':
+        if name in phone_book:
+           print(f"{name} is exist, please enter another name")
+        else:
+            phone_book[name] = number
+            print (phone_book.keys())
```

Comment:
Here you also need to update your file by dumping the `phone_book` to a JSON string (json.dumps) and writing the string to a file by rewriting the file

Reply:
seems like done